astronomer/providers/google/cloud/example_dags/example_dataproc.py

```diff
@@ -198,14 +199,17 @@
     )
     # [END howto_delete_buckettask]
 
-    create_cluster >> update_cluster >> create_bucket
-    create_cluster >> update_cluster >> pig_task >> hive_task >> delete_cluster >> delete_bucket
+    end = EmptyOperator(task_id="end")
+
+    create_cluster >> update_cluster >> hive_task >> spark_task >> spark_sql_task >> delete_cluster
```

Comment:
Is spark_task dependent upon hive_task? If not, it can run in parallel to the hive_task.

Also, previously it looks like hive_task is dependent on pig_task. Is that not a case? @rajaths010494 can you confirm?

Reply:
No it's not dependent we can run in parallel. Hive task is dependent on creating a bucket

Reply:
```
HIVE_JOB = {
    "reference": {"project_id": PROJECT_ID},
    "placement": {"cluster_name": CLUSTER_NAME},
    "hive_job": {"query_list": {"queries": ["SHOW DATABASES;"]}},
}
```

Reply:
This looks good 👍 .